src/core/components/start.js

```diff
@@ -2,11 +2,13 @@
 
 const series = require('async/series')
 const Bitswap = require('ipfs-bitswap')
+const get = require('lodash/get')
```

Comment:
use smaller package 
https://bundlephobia.com/result?p=lodash.get@4.4.2
vs
https://bundlephobia.com/result?p=dlv@1.1.2
```suggestion
const get = require('dlv')
```

Reply:
Cool, thanks! Will add `dlv` to `package.json` as well.

Reply:
I'm fine with using `dlv` over `loadash.get` but all other occurances of `lodash.get` need to be replaced as well otherwise we're just adding more to our bundle.

Reply:
If you agree, I can create other PR with all those occurrences changed to `dlv`.